web/src/components/PrivacyPolicy.vue

```diff
@@ -0,0 +1,12 @@
+<template>
+  <div class="prose max-w-prose">
```

Comment:
tailwind stuff?

Reply:
Yeah, just nice formatting of plain HTML markdown
https://tailwindcss.com/docs/typography-plugin